nixos/tests/kernel-generic.nix

```
@@ -7,6 +7,89 @@
 with pkgs.lib;
 
 let
+  patchedPkgs = pkgs.extend (
+    final: prev: {
+      linuxKernel = prev.linuxKernel // {
+        kernelPackagesExtensions = prev.linuxKernel.kernelPackagesExtensions ++ [
+          (
+            finalKernelPackage: _:
```

```suggestion
            finalKernelPackages: _:
```

Because it's the package set, right?